packages/perseus/src/widgets/number-line/number-line.tsx

```diff
@@ -124,17 +124,16 @@ const TickMarks: any = Graphie.createSimpleClass((graphie, props) => {
     const results: Array<any> = [];
 
     // For convenience, extract some props into separate variables
-    const range = props.range;
-    const labelRange = props.labelRange;
+    const {range, labelRange, labelStyle, labelTicks, tickStep} = props;
```

Comment:
Minor: why not destructure `numDivisions` along with the other props?

(I think adding `numDivisions` here would cause Prettier to break this statement into multiple lines, which... might not be worth the consistency. But I'm throwing it out there as an option.)

Reply:
Good catch.